my-app/META-INF/RASCAL.MF

```diff
@@ -1,5 +1,6 @@
 Manifest-Version: 0.0.1
 Main-Function: main
 Project-Name: my-app
+Require-Libraries: |file:////Users/auke/.m2/repository/frinklang/frink/22.4.18/frink-22.4.18.jar|
```

Comment:
Specify the full path of the installed `.jar` to ensure that Rascal can find it.